ReadMe.md

```diff
@@ -193,7 +227,7 @@ As it's an Agile Methodology, we're hosting weekly Live Streams and assigning th
 
 #### Can I work on an already opened issue?
 
-You can always ask in the comments section of the opened issue to get yourself assigned but the owner may or may not assign you that issue. However, the owner may suggest you a fresh issue.
+You can always ask in the comments section of the opened issue to get yourself assigned but the owner may or may not assign you that issue. However, the owner may suggest you a fresh issue (in which you have to create new issue by reading about that Project,searchig the bug present in it).
```

Comment:
Grammar changes, spelling mistakes to be corrected please. 😊

Reply:
sure sir.